x11-misc/adriconf/metadata.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>desktop-misc@gentoo.org</email>
+		<name>Gentoo Desktop Miscellaneous Project</name>
+	</maintainer>
```

Comment:
`desktop-misc` project has been disbanded. See https://marc.info/?l=gentoo-dev&m=160642206127820&w=4

Reply:
hi @Polynomial-C can you take care of a similar PR?
I don't know how to continue.

I don't know how to give you credit in this PR. All this code was copied from your overlay.

thanks :)

Reply:
I would be willing to steppin as maintainer.